### DISCUSSION TITLE
Hi all I recently encountered a hardware [Feb 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5088] [Discord Threads]
> 🙋‍♂️ **DungeonSmasher**

**Comment by o1lab** [comment]
> 🙋‍♂️ **davidpfahler**  
#### Currency is still available in latest release. My guess is that something screwed up in the restore and now you have some kind of half-done state in your database.  
#  
> 🙋‍♂️ **DungeonSmasher**  
#### i wouldn't rule it out for sure, are there logs I can check our export this project and re-import it to repair it?  
#  
> 🙋‍♂️ **davidpfahler**  
#### I wouldn't know, but maybe someone from the <@&682859083799199814>  can help? I had a database issue once when updating from one version to another and fixed it by directly connecting to the underlying postgres database, but that is bombad dangerous, so you better know what you're doing.  
#  
> 🙋‍♂️ **DungeonSmasher**  
#### Based on your guess I began preparations to migrate the data to a new project. When inspecting the problem field, I noticed that the type had changed from currency to `SpecificDBType` I changed it back and the field now appears on the form. So now I can at least get the form back up while i migrate it 😄   
#### Thanks for your suggestion!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1072161154887073792)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
